apps/vite/src/components/calendar/TodayEventList.tsx

```
@@ -0,0 +1,53 @@
+import nepaliNumber from "@/helper/nepaliNumber"
+import { cn } from "@/lib/utils"
+import { EventDetail } from "@miti/types"
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)